### DISCUSSION TITLE
ignore build step

### DISCUSSION THREAD

**Discussion by Yassin-Samir** [vercel/community#6319] [Help] [answered]
### Question

i want to create an ignore build step script to build only when it is production not preview and when only certain directories change

**Comment by amyegan** [comment, marked as answer]
Hey @Yassin-Samir. You can use a custom [Ignored Build Step](https://vercel.com/docs/projects/overview#ignored-build-step) script to accomplish this. 

Some example scripts are available in this guide: https://vercel.com/guides/how-do-i-use-the-ignored-build-step-field-on-vercel

And available System Environment Variables are listed here: https://vercel.com/docs/projects/environment-variables/system-environment-variables#system-environment-variables
